src/GitHub/GraphQL.hs

```diff
@@ -84,16 +88,63 @@ data ParamName
     | ParamName
     | ParamLast
     | ParamStates
+    | ParamOrderBy
+    | ParamField
+    | ParamDirection
     deriving stock (Show)
 
 data ParamValue
+    {- | Textual parameter:
+
+    @
+    name: "github-graphql"
+    @
+    -}
     = ParamStringV !Text
+
+    {- | Integer parameter:
+
+    @
+    last: 2
+    @
+    -}
     | ParamIntV !Int
+
+    {- | Issue/PR states:
+
+    @
+    states: [CLOSED, MERGED]"
+    @
+    -}
     | ParamStatesV !(NonEmpty State)
+
+    | ParamIssueOrderField !IssueOrderField
+
+    | ParamOrderDirection !OrderDirection
```

Comment:
I've added documentation to some fields. But then realized that there're too much enums in GitHub API, so would be nice to do something with having a lot of enums... If you think that such docs are great, I can add them everywhere

Reply:
No, I don't have anything particular in mind... So let's go with constructors for enums for now.
But I'll add the docs.